app.py

```diff
@@ -4,6 +4,7 @@
 
 app = Flask(__name__)
 app.config['SQLALCHEMY_DATABASE_URI'] = os.environ['DATABASE_URL']
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False # Suppress a warning message
```

Comment:
You'll want this to be `os.environ['SQLALCHEMY_TRACK_MODIFICATIONS']` rather than `false`, in order for your previous commit to have effect.